packages/server/routes/api/auth.js

```diff
@@ -107,4 +114,86 @@ router.get("/google/callback", passport.authenticate("google", {
   response.redirect(`icebreak://login?token=${newToken}`);
 });
 
-module.exports = router;
\ No newline at end of file
+router.post("/register", async (request, response) => {
+
+  try{
+    const { email, password, } = request.body;
+    
+    if(!email.includes("@") || email.includes(" ")){ // check if email is valid
+      throw new Error("Email is invalid.");
+    }
+
+    const ifExist = await user.getUserByEmail(email);
+
+    if(ifExist?.email == email){ // check if email is already in the database
```

Comment:
Extremely nitpicky and you don't have to change it, but you should use === to check for both type and value.

Reply:
No, I highly recommend to change it to reduce unexpected bugs with types. It's always best to use strictly equally whenever you can.